### PR TITLE
feat: show passcode asap

### PR DESCRIPTION
**Pull request by jia1**
## Problem

Closes [SGC-209](https://linear.app/ogp/issue/SGC-209/show-passcode-immediately)

## Notes

- Remember to run pending migrations in the various envs

### CODE REVIEW

File: backend/src/govsg/services/govsg.service.ts [hunk 1]
```diff
@@ -170,20 +172,35 @@ export function uploadCompleteOnChunk({
...
+            })
+          }
+        },
+        benchmark: true,
+        returning: true,
+      }
+    )
+    await Promise.all(
+      govsgMessages.map((message) =>
+        GovsgVerification.create(
```
Comment (KishenKumarrrrr) [inline backend/src/govsg/services/govsg.service.ts:195]: Could I ask why we can't make use of `bulkCreate` here?

**Review by KishenKumarrrrr — APPROVED** [review]
Just some small comments, but otherwise LGTM

File: backend/src/govsg/services/govsg.service.ts (hunk 1 above)
Comment (jia1) [reply backend/src/govsg/services/govsg.service.ts]: Cus I was silly. Using `bulkCreate` now!

File: backend/src/govsg/services/govsg-verification-service.ts [hunk 2]
```diff
@@ -31,20 +30,23 @@ export const sendPasscodeCreationMessage = async (
...
-}
-
 export const storePrecreatedPasscode = async (
   govsgMessageId: GovsgMessage['id'],
   passcodeCreationWamid: MessageId
 ): Promise<GovsgVerification> => {
-  const passcode = createPasscode()
-  return await GovsgVerification.create({
-    govsgMessageId,
+  const govsgVerification = await GovsgVerification.findOne({
```
[the remark this answers is not in the document]
Comment (jia1) [reply backend/src/govsg/services/govsg-verification-service.ts:37]: To the best of my knowledge, `upsert` requires the knowledge of the entire row's data / identifier. There's no unique constraint on the field I'm searching with.

**Pull request merged**